site/content/kapp/docs/latest/diff.md

```diff
@@ -30,8 +30,48 @@ Anytime there is a change to a resource marked as a versioned resource, entirely
 
 To make resource versioned, add `kapp.k14s.io/versioned` annotation with an empty value. Created resource follow `{resource-name}-ver-{n}` naming pattern by incrementing `n` any time there is a change.
 
+Example:
+```
```

Comment:
```suggestion
```yaml
```
Doing something like this should let us get some syntax highlighting

Reply:
We could do this for both YAML examples. We could use ````zsh` for the terminal examples in a similar manner but it might not help a lot

Reply:
use ```bash for terminal

Reply:
Done